jsapp/xlform/src/model.configs.coffee

```diff
@@ -218,7 +218,7 @@ module.exports = do ->
       value: ""
       _hideUnlessChanged: true
     required:
```

Comment:
Biggest change in this PR is this (and corresponding test changes)

New questions will no longer have `"required=true"` when the model is initialized.